src/main/java/com/minecolonies/coremod/client/gui/WindowHutDeliveryman.java

```diff
@@ -1,14 +1,15 @@
 package com.minecolonies.coremod.client.gui;
 
 import com.minecolonies.blockout.controls.Button;
+import com.minecolonies.blockout.controls.ButtonHandler;
 import com.minecolonies.coremod.colony.buildings.BuildingDeliveryman;
 import com.minecolonies.coremod.lib.Constants;
 import org.jetbrains.annotations.NotNull;
 
 /**
  * Window for the farmer hut.
  */
-public class WindowHutDeliveryman extends AbstractWindowWorkerBuilding<BuildingDeliveryman.View> implements Button.Handler
+public class WindowHutDeliveryman extends AbstractWindowWorkerBuilding<BuildingDeliveryman.View> implements ButtonHandler
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') "ButtonHandler" is implemented by a super class; there is no need to implement it here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1939)